content/07.pathogenesis.md

```diff
@@ -55,13 +55,19 @@ They are non-segmented, which means the viral genome is contained in a single ca
 Coronaviruses have large positive-sense RNA (ssRNA+) genomes ranging from 27 to 32 kilobases in length [@doi:10.1007/978-1-4939-2438-7_1; @doi:10/ggjr43].
 The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp [@doi:10/ggjr43].
 Genome organization is highly conserved within the order [@doi:10.1007/978-1-4939-2438-7_1].
-There are three major regions: one containing the replicase gene and one containing the genes encoding structural proteins [@doi:10.1007/978-1-4939-2438-7_1].
+There are three major regions: one containing the replicase gene and one containing the genes encoding structural proteins [@doi:10.1007/978-1-4939-2438-7_1] (Figure @fig:genome-structure).
 The replicase gene comprises about two-thirds of the genome and consists of two open reading frames that are translated with ribosomal frameshifting [@doi:10.1007/978-1-4939-2438-7_1].
 This polypeptide is then translated into 16 non-structural proteins (nsp), except in Gammacoronaviruses where nsp1 is absent, that form replication machinery used to synthesize viral RNA [@doi:10.1002/jmv.25681].
 The remaining third of the genome encodes structural proteins, including the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins.
-Additional accessory genes are sometimes present between these two regions, depending on the species or strain. <!-- TODO: Add a figure showing the structure of the genome.-->
+Additional accessory genes are sometimes present between these two regions, depending on the species or strain.
 Much attention has been focused on the S protein, which is a critical structure involved in cell entry.
 
+![**Structure of SARS-CoV-2 capsid and genome.**
+A) The genomic structure of coronaviruses is highly conserved, including the order and organization of genes such as the spike (S), envelope (E), membrane (M) and nucleocapsid (N) proteins.
+B) The physical structure of the coronavirus virion, including the components encoded by conserved genes S, E, M and N. 
+](images/genome-structure.png)
+{#fig:genome-structure secno=1}
```

Comment:
```suggestion
](images/genome-structure.png){#fig:genome-structure secno=1}
```

Something still isn't working with this figure, but I don't know what's wrong.  This is a shot in the dark.

I also noticed pandoc-fignos is complaining that fig:csse-deaths and fig:overview are defined multiple times.  I don't think that is causing this issue but will need to be addressed separately.